user_tools/src/spark_rapids_pytools/cloud_api/databricks_aws.py

```diff
@@ -31,6 +31,7 @@
 from spark_rapids_pytools.pricing.price_provider import SavingsEstimator
 
 
+# pylint: disable=abstract-method
```

Comment:
QQ: What prompt that change in all the cloud_api classes??

Reply:
Addition of method `create_distributed_submission_job()` in `PlatformBase` required all CSPs to implement this. Now, since CSPs do not support distributed mode now, we would have to implement this method in all CSPs modules with body as `pass`. 

Currently, we use the above approach for methods such as:

```python
def set_offline_cluster(self, cluster_args: dict = None):
   pass
        
def validate_job_submission_args(self, submission_args: dict) -> dict:
   pass
```

However, I think there are pros and cons to this approach.

Pros: Each CSP class we are clear what is implemented, what is not.
Cons: It adds redundant code in all CSP classes.

By adding the pylint exception, it would not be mandatory for each CSP to define methods with body as `pass`. Let me know your thoughts on this.

Reply:
From offline discussion, removed the disable rule for pylint and added `create_distributed_submission_job()` in each CSP.